opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -100,7 +99,8 @@ impl LogProcessor for SimpleLogProcessor {
         // noop after shutdown
         if self.is_shutdown.load(std::sync::atomic::Ordering::Relaxed) {
             otel_warn!(
-                name: "simple_log_processor_emit_after_shutdown"
+                name: "SimpleLogProcessor.Export.AfterShutdown",
+                error = LogError::Other("Attempted to export a log after processor shutdown".into())
```

Comment:
is there any advantage of wrapping this into LogError?

Reply:
I find it useful - it adds convenience on top of  [std::error::Error](https://doc.rust-lang.org/std/error/trait.Error.html), by declaring various error types as enum and mapping it to actual error string. We don't need to add the error string while returning the error.